code/datums/jobs.dm

```diff
@@ -725,11 +725,12 @@
 	wages = 150
 
 	slot_back = /obj/item/storage/backpack/withO2
-	slot_belt = /obj/item/device/pda2/mechanic
+	slot_belt = /obj/item/storage/belt/utility/prepared
 	slot_jump = /obj/item/clothing/under/rank/mechanic
 	slot_foot = /obj/item/clothing/shoes/black
 	slot_lhan = /obj/item/storage/toolbox/electrical/mechanic_spawn
```

Comment:
Probably want to remove the duplicate tools from the toolbox the mechanic's spawn with?

Reply:
Good shout, will follow up with some adjustments